app/(public)/components/paragraph/sul-contact-card/node-reference-card.tsx

```diff
@@ -4,31 +4,26 @@ import Link from "next/link";
 import Image from "next/image";
 import {ClockIcon, EnvelopeIcon, MapPinIcon, PhoneIcon } from "@heroicons/react/24/outline";
 import useLibraryHours from "@/lib/hooks/useLibraryHours";
-import Conditional from "../utils/conditional";
+import Conditional from "@/components/utils/conditional";
 import {useResizeDetector} from "react-resize-detector";
 import { ContactCardParagraph } from "@/lib/drupal/drupal";
 
-interface ContactCardProps {
-  paragraph: ContactCardParagraph
-  siblingCount?: number
-}
-
-const SulContactCard = ({paragraph, siblingCount}: ContactCardProps) => {
+const NodeReferenceCard = ({paragraph}: { paragraph: ContactCardParagraph }) => {
   const {width, ref} = useResizeDetector();
   const hours = useLibraryHours()
 
-const toISOStringWithTimezone = date => {
-  const tzOffset = -date.getTimezoneOffset();
-  const diff = tzOffset >= 0 ? '+' : '-';
-  const pad = n => `${Math.floor(Math.abs(n))}`.padStart(2, '0');
-  return date.getFullYear() +
-    '-' + pad(date.getMonth() + 1) +
-    '-' + pad(date.getDate()) +
-    'T' + pad(date.getHours()) +
-    ':' + pad(date.getMinutes()) +
-    ':' + pad(date.getSeconds()) +
-    diff + pad(tzOffset / 60) +
-    ':' + pad(tzOffset % 60);
+  const toISOStringWithTimezone = date => {
```

Comment:
Did you get this from somewhere?

Reply:
[Yeah](https://www.30secondsofcode.org/js/s/to-iso-string-with-timezone), I grabbed it to fix timezone issues with `.toISOString` - the current day being calculated was off, so it would switch to the next day at the wrong time. I'm seeing `.toISOString` in your `today-hours.tsx` file for the homepage banner, so I'm wondering if there are issues there as well.

Reply:
We might want to put this into it's own function so that it can be shared. I might move it when I fix the events display.